ansible/roles/debops.kmod/tasks/main.yml

```diff
@@ -60,5 +61,4 @@
 
 - name: Update Ansible facts if the list of load kernel modules might have changed
   action: setup
-  when: (kmod__register_module_config_delete is changed or
-         kmod__register_module_config_create is changed)
+  when: kmod__register_modprobe_tasks is changed
```

Comment:
Just to be sure I understand this correctly - the outer loop task will record the results of inner loops... Will it have a `changed` state for all of the results, even if some of them later on are not changed? Just want to make sure it works as expected, I don't think I ever registered an outer loop like this.

Reply:
Damn, I was wrong again. I played with it a bit more now and it seems there is no easy way to run a task when one or more tasks in included tasks have the "changed" state. I am changing it back in #263.

Reply:
As I suspected. Thanks for checking. :-)